dandiapi/api/migrations/0031_asset_asset_metadata_has_schema_version_and_more.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.0.6 on 2022-07-18 01:46
+
+from django.conf import settings
+from django.db import migrations, models, transaction
+
+
+def populate_empty_asset_schema_versions(apps, _):
+    Asset = apps.get_model('api', 'Asset')
+
+    with transaction.atomic():
+        for asset in Asset.objects.filter(metadata__schemaVersion__isnull=True).iterator():
+            asset.metadata.setdefault('schemaVersion', settings.DANDI_SCHEMA_VERSION)
+            asset.save(update_fields=['metadata'])
```

Comment:
I don't see this being done with version metadata anywhere, shouldn't that also be added?

Reply:
The API always populates the schemaVersion when a dandiset is created (https://github.com/dandi/dandi-archive/blob/master/dandiapi/api/views/dandiset.py#L246), so it's less likely to be missing a schemaVersion (the only scenario I believe, is if someone did a PUT to an existing version's metadata with it missing).

I double checked, and `Version.objects.filter(metadata__schemaVersion__isnull=True).exists()` returns false on both production and staging.